Shared_AIHS2/Clothes/KoiClothesOverlayController.Hooks.cs

```diff
@@ -38,7 +41,7 @@ public static void ChangeCustomClothesPostHook(ChaControl __instance, int kind)
                 // Clean up no longer used textures after some clothes slots get disabled
                 if (MakerAPI.InsideMaker && controller.CurrentOverlayTextures != null)
                 {
-                    var toRemoveList = controller.CurrentOverlayTextures.Where(x => !x.Value.IsEmpty() && controller.GetCustomClothesComponent(x.Key) == null).ToList();
+                    var toRemoveList = controller.CurrentOverlayTextures.Where(x => !x.Value.IsEmpty() && controller.GetCustomClothesComponent(GetRealClothesId(x.Key)) == null).ToList();
```

Comment:
This should be name just `GetRealId` because as it is now it looks like it may be unintended/mistake to get clothes ID here.